src/components/books/books.component.js

```diff
@@ -0,0 +1,23 @@
+const angular = require('angular');
+
+angular.module('app-bootstrap')
+  .component('books', {
+    template: require('./books.html'),
+    controller: ['booksService', function (booksService) {
+      this.bookCover = require('./../../assets/img/book-cover.png');
+      this.books = booksService.getBooksList();
+    }]
+  })
+  .filter('searchBook', function() {
+    return function (items, searchText) {
+      let filtered = [];
+      let letterMatch = new RegExp(searchText, 'i');
+      for (var i = 0; i < items.length; i++) {
+        var item = items[i];
+        if (letterMatch.test(item.title) || letterMatch.test(item.author)) {
+          filtered.push(item);
+        }
+      }
```

Comment:
I think there's a better way to do this. Check the `filter` function and the `includes` function 😉